guide/using-visual-cpp-with-webots.md

```diff
@@ -142,25 +142,40 @@ Now the controller should pause when it reaches one of your breakpoints.
 
 Webots contains several `C` or `C++` libraries based on the `libController` or
 `libCppController` libraries (e.g. the `automobile library`, `DARwIn-OP library`,
-etc.).
-The corresponding precompiled libraries are incompatible with Visual C++,
-because they are compiled using the `gcc` compiler tool chain embedded in Webots.
+`youBot library`, etc.).
 
-To use these libraries with your project, their source files should be compiled directly in your project,
-exactly as it's done for the `libCppController` library (cf. instructions above).
+The precompiled `C` libraries are released with the corresponding Visual Studio `.lib` file
+(the linker to the `.dll` file) to facilitate their integration into a Visual Studio project.
+There is one `.lib` file per architecture (`.x32.lib` for the 32 bits architecture
+and `.x64.lib` for the 64 bits architecture).
```

Comment:
Yes, exactly. This is more aligned with what we have been doing so far with Controller.lib and it follows the msys64 files organization rules.